jenkins/sars_cov_2.py

```diff
@@ -47,33 +38,3 @@ def get_expected_output_files(output_path: str, sample_ids: List[str], sequencin
     output_files.append(join_path(output_path, "resultfiles.json"))
 
     return output_files
-
-
-@click.command(name="sars_cov_2")
```

Comment:
Here I changed the logic because this code is identical to the synthetic pipeline except for `get_expected_output_files()`. Also, the argument `--sequencing-technology` is not used for the synthetic pipeline as the generated output files are the same independently of illumina or ont processed samples.